tests/func/test_remote_dir.py

```diff
@@ -1,12 +1,25 @@
 # -*- coding: utf-8 -*-
```

Comment:
This is still units tests since we don't make a repo and don't start all the dvc machinery.

Reply:
Agreed.